src/mixpanelMiddleware.js

```diff
@@ -46,6 +47,10 @@ export default function mixpanelMiddleware(token, options = {}) {
             });
         }
 
+        if (timeEvent && timeEvent.length > 0) {
+            mixpanel.time_event(timeEvent);
```

Comment:
It looks like `time_event` also accepts a config object as the second parameter, maybe we should handle this too?